_posts/2022-06-09-fivetran-webhook-notifications.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title: Fivetran Slack Notifications
+date: 2022-06-09
+tags: fivetran slack notifications incident-management webhook 
+author: Joseph Miocevich
+image: img/fivetran_logo.png
+description:
+  Creating fivetran event notifications using webhooks
+---
+
+![Fivetran Slack Notifications](/img/fivetranslacklambdalogo.png)
+
+<br/>
+
+Getting timley notifications on whether a Fivetran data connector is broken, or a dbt transformation has failed, is crucial to maintaining confidence in your datapipeline and reducing your time to restore. This blog will cover how to build your own system in AWS utilising Fivetran's newly released [webhooks](https://fivetran.com/docs/rest-api/webhooks) functionality and [slack webhooks](https://slack.com/intl/en-au/help/articles/115005265063-Incoming-webhooks-for-Slack).
```

Comment:
perhaps link dbt to site so the user knows what it is..

Reply:
```suggestion
Getting timley notifications on whether a Fivetran data connector is broken, or a [dbt transformation](https://fivetran.com/docs/transformations/dbt) has failed, is crucial to maintaining confidence in your datapipeline and reducing your time to restore. This blog will cover how to build your own system in AWS utilising Fivetran's newly released [webhooks](https://fivetran.com/docs/rest-api/webhooks) functionality and [slack webhooks](https://slack.com/intl/en-au/help/articles/115005265063-Incoming-webhooks-for-Slack).
```